people/matej.json

```diff
@@ -0,0 +1,11 @@
+{
+  "key": "matej",
+  "name": "Matěj Klouček",
+  "nick": "Matěj",
+  "email": "matej.kloucek@mild.blue",
+  "title_cs": "JUNIOR DEVELOPER",
+  "title_en": "JUNIOR DEVELOPER",
+  "short_desc_en": "Matěj studies Mathematical Informatics at the Faculty of Nuclear Sciences and Physical Engineering at CTU. At Mild Blue, he works as a junior developer. His interests include graph theory, neural networks and optimalizazion. In his spare time Matěj loves to listen to music, travel or just hang around with friends.",
```

Comment:
In his spare time Matěj loves to listen to music, travel or just hang around with friends.

->

When he is not working or studying, Matěj loves listening to good music, travelling or just hanging out with friends.